server/src/db/chunk.rs

```diff
@@ -432,7 +432,11 @@ impl QueryChunk for DbChunk {
         }
     }
 
-    // TODOs: return the right value. For now the chunk is assumed to be not sorted
+    /// Returns true if the chunk is sorted on its pk
+    /// Since data is compacted prior being moved to RUBs, data in RUBs and OBs
```

Comment:
I am not sure what a `OBs` means

Reply:
I think @NGA-TRAN means Object Stores?

```suggestion
    /// Since data is compacted prior being moved to RUBs, data in RUBs and object store
```